src/appkit.cs

```diff
@@ -22287,8 +22290,10 @@ partial interface NSView {
 	[Category, BaseType (typeof (NSApplication))]
 	partial interface NSRemoteNotifications_NSApplication {
 
+#if !XAMCORE_4_0
```

Comment:
Probably put it around the whole NSRemoteNotifications_NSApplication interface (and attributes) since this is the only thing in it.